circle.yml

```diff
@@ -44,3 +44,9 @@ test:
   post:
     - killall --wait sc:  # wait until Sauce Connect closes the tunnel
         parallel: true
+
+deployment:
+  master:
+    branch: master
+    commands:
+      - ssh -l openfisca fr.openfisca.org "cd fr.openfisca.org; ./deploy.sh"
```

Comment:
> Why not using a [deploy user](https://gist.github.com/MattiSG/3a5e106449a0786e4454d11338633bcd) like we usually do ?  (`ssh deploy-fr.openfisca.org@fr.openfisca.org`) ?
> With this, we have to give a key that is able to do a lot of things on the server, which is less safe that a deploy user.

Reply:
This is -or seems to be- the way `legislation-explorer` is deployed. I haven't find a way to make `deploy-fr.openfisca.org` do `npm install` or `nom run build`. If know how to make it work, I'm very interested by the solution!

> `fr.openfisca.org` is in the `openfisca` user account. In this context, `deploy-fr.openfisca.org` wouldn't have access to  `/node_modules` & `/.next` directories (`/home/openfisca/fr.openfisca.org/node_modules/`, ...). So, it wouldn't be able to `npm install` and `npm run build`.

Reply:
> deploy-fr.openfisca.org do npm install or nom run build

You just tell `deploy-fr.openfisca.org` which script to run, and the script contain all the instructions you want.

> fr.openfisca.org is in the openfisca user account. (...)

Yep, but there is a trick : the first line of the deployment script is `su openfisca` 🙂

Reply:
Ok to wait for another PR though.